pkg/controller/certificaterequest/certificaterequest_controller.go

```diff
@@ -141,6 +144,7 @@ func (r *ReconcileCertificateRequest) Reconcile(request reconcile.Request) (reco
 	// Add finalizer if not exists
 	if !utils.ContainsString(cr.ObjectMeta.Finalizers, certmanv1alpha1.CertmanOperatorFinalizerLabel) {
 		reqLogger.Info("adding finalizer to the certificate request")
+		localmetrics.IncrementCertRequestsCounter()
```

Comment:
If the operator pod is restarted and there are some existing `CertfificateRequest` objects then they will get counted twice. Once during the `CheckInitCounter` and then while adding the finalizer.

Reply:
Yes, that's why I put the limitation in the describtion of the Init method 
``` 
 Current version does not support well multiple instances of the operator to run on the same Hive cluster 
```
We discussed it with James and thought it would be OK for a start.

Reply:
This will affect the values returned even when there is a single pod. say there are bunch of certificate requests which are created. the operator starts dealing with them but crashes without adding finalizers. when it restarts it counts all of them and then when it adds the finalizers it counts them again.

Reply:
ah yes, thanks.. Will filter to count only the ones having the finalzer